bluepill/bluepill.xcodeproj/project.pbxproj

```diff
@@ -376,9 +378,11 @@
 			buildSettings = {
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				DEVELOPMENT_TEAM = 57Y47U492U;
+				DSTROOT = "";
 				FRAMEWORK_SEARCH_PATHS = /Library/Developer/PrivateFrameworks/;
 				HEADER_SEARCH_PATHS = "\"$(SRCROOT)/..\"";
 				INFOPLIST_FILE = tests/Info.plist;
+				INSTALL_ROOT = /;
```

Comment:
are these needed?

Reply:
These are needed or there will be build error for Xcode 15